nx-X11/programs/Xserver/randr/rrcrtc.c

```diff
@@ -174,16 +198,24 @@ RRCrtcNotify(RRCrtcPtr crtc,
 
         if (numOutputs) {
             if (crtc->numOutputs)
+#ifndef NXAGENT_SERVER
+                newoutputs = reallocarray(crtc->outputs,
+                                          numOutputs, sizeof(RROutputPtr));
```

Comment:
Why not provide helper functions for (re)allocarray for now? This would save a couple of ifndefs...